R/pkg/R/functions.R

```diff
@@ -1730,24 +1730,82 @@ setMethod("toRadians",
 
 #' to_date
 #'
-#' Converts the column into DateType.
+#' Converts the column into DateType, optionally with a format. Returns null if input is invalid.
+#' Specify formats according to: http://docs.oracle.com/javase/tutorial/i18n/format/simpleDateFormat.html
 #'
-#' @param x Column to compute on.
+#' @param x Column to parse.
+#' @param format string to use to parse x Column to DateType.
 #'
 #' @rdname to_date
 #' @name to_date
 #' @family datetime_funcs
 #' @aliases to_date,Column-method
 #' @export
-#' @examples \dontrun{to_date(df$c)}
-#' @note to_date since 1.5.0
+#' @examples
+#' \dontrun{
+#' to_date(df$c)
+#' to_date(df$c, 'yyyy-MM-dd')
+#' }
+#' @note to_date(Column, format) since 2.2.0
```

Comment:
this should be the type, not parameter name; so it should be
`#' @note to_date(Column, character) since 2.2.0`
ditto below